### PR TITLE
Improve a11y document and dropdown item

### PR DESCRIPTION
**Pull request by wxiaoguang**

### CODE REVIEW

File: web_src/js/modules/fomantic/aria.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 This document is used as aria/accessibility(a11y) reference for future developers.
 
-There are a lot of a11y problems in the Fomantic UI library. This `aria.js` is used
-as a workaround to make the UI more accessible.
+There are a lot of a11y problems in the Fomantic UI library. Some code in GItea's own
```
Comment (lunny) [inline web_src/js/modules/fomantic/aria.md]:
```suggestion
There are a lot of a11y problems in the Fomantic UI library. Some code in Gitea's own
```

**Review by lunny — APPROVED** [review]

**Comment by wxiaoguang** [comment]
> Is there some credit that should be added here?

Feel free to edit this PR directly.

**Comment by wxiaoguang** [comment]
> The fix for this seems similar to https://codeberg.org/forgejo/forgejo/pulls/2636
> 
> Is there some credit that should be added here?

By taking a deep look, I do not think there is a strong requirement for either side. Because for example: https://codeberg.org/forgejo/forgejo/pulls/2642  , this has been fixed on Gitea side days ago.

File: web_src/js/modules/fomantic/aria.md [hunk 2]
```diff
@@ -2,10 +2,10 @@
 
 This document is used as aria/accessibility(a11y) reference for future developers.
 
-There are a lot of a11y problems in the Fomantic UI library. This `aria.js` is used
-as a workaround to make the UI more accessible.
+There are a lot of a11y problems in the Fomantic UI library. Some code in Gitea's own
+"js/modules/fomaintic/*.js" files is used as a workaround to make the UI more accessible.
```
Comment (silverwind) [inline web_src/js/modules/fomantic/aria.md]:
```suggestion
Files in `web_src/js/modules/fomantic` are used as a workaround to make the UI more accessible.
```

File: web_src/js/modules/fomantic/aria.md [hunk 3]
```diff
@@ -2,10 +2,10 @@
 
 This document is used as aria/accessibility(a11y) reference for future developers.
 
-There are a lot of a11y problems in the Fomantic UI library. This `aria.js` is used
-as a workaround to make the UI more accessible.
+There are a lot of a11y problems in the Fomantic UI library. Some code in Gitea's own
+"js/modules/fomantic/*.js" files is used as a workaround to make the UI more accessible.
```
Comment (silverwind) [inline web_src/js/modules/fomantic/aria.md]:
```suggestion
There are a lot of a11y problems in the Fomantic UI library. Files in 
`web_src/js/modules/fomantic/` are used as a workaround to make the UI more accessible.
```

**Review by silverwind — APPROVED** [review]

**Comment by oliverpool** [comment]
> Because for example: https://codeberg.org/forgejo/forgejo/pulls/2642 , this has been fixed on Gitea side days ago.

@wxiaoguang if you read your counter-example to the end you can see that:
- I didn’t saw it was fixed in Gitea when I opened the PR
- As soon as I became aware, I closed my PR in favor of the cherry-pick (which preserved the original commit, so full attribution to the committer)

**Comment by wxiaoguang** [comment]
Thank you for you explanation.

And this PR was started for https://github.com/go-gitea/gitea/pull/29400#issuecomment-1982106870 to address my concern for the new "menu" (Mar 7, 2024). I reviewed the documents and the dropdown implementation and improved them.

**Pull request merged**